pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,128 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+def test_spectral_factor_pelland(airmass_absolute, clearness_index,
```

Comment:
@RDaxini sorry I wasn't clear. Use 'test_' as the first part of function names in the 'test_spectrum.py' fiel that pytest processes. Here, in mismatch.py, we don't put 'test_' in the function name.